.github/workflows/pull_request.yml

```diff
@@ -58,10 +58,26 @@ jobs:
       with:
         # needs depth to run git log below
         fetch-depth: 50
+    - uses: actions/cache@v4
+      id: cache
+      with:
+        path: ./build-cache
+        key: taxonomies-${{ hashFiles('taxonomies/**') }}
+        restore-keys: taxonomies-
+    - if: steps.cache.outputs.cache-hit == 'true'
+      name: Prepare build_cache volumes
+      run: |
+        proj=po_off
+        docker volume create \
```

Comment:
Would it be possible to use something like [--opt device=./build-cache](https://docs.docker.com/reference/cli/docker/volume/create/#opt), i.e. mount the folder directly, to avoid having to copy files to and from the docker volume?

Reply:
done :smile: